src/pages/ValidateConfigFiles.js

```diff
@@ -140,14 +140,29 @@ class DisplayFileList extends Component {
         </div>);
     } else if (this.props.valid === VALIDATING) {
       return (
-        <div> <i className='material-icons refresh-icon'>refresh</i>
-          {translate('validate.config.files.msg.validating')}</div>
+        <div>
+          <i className="eos-icons eos-icon-loading mr-3"></i>
+          {translate('validate.config.files.msg.validating')}
+        </div>
       );
     } else if (this.props.valid === VALID) {
-      return (<div>{translate('validate.config.files.msg.valid')}</div>);
+      return (
+        <Alert key='validate-success' variant='success'>
+          {translate('validate.config.files.msg.valid')}
+        </Alert>
+      );
     } else {
-      return (<div>{translate('validate.config.files.msg.invalid')}<br/>
-        <pre className='log'>{this.props.invalidMsg}</pre></div>);
+      return (
+        <Alert key='validate-invalid' variant='danger'>
+          <div>
+            {translate('validate.config.files.msg.invalid')}
+            <br/>
+            <pre className='log'>
+              {this.props.invalidMsg}
+            </pre>
+          </div>
+        </Alert>
```

Comment:
The effect looks a little bit odd with red border with white log contents. Not sure it is expected.